projects/common-components/src/lib/button/button.component.scss

```diff
@@ -42,3 +42,34 @@
 		@include bottom-right(57%, -10%);
 	}
 }
+
+:host.submit .button{
```

Comment:
Is it needed here **.button** if it define defined above?

Reply:
Yes it's necessary, I tried without this it's doesn't work

Reply:
try this:

:host(.submit) {
  .button {...}
}